src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/InheritDocTaglet.java

```diff
@@ -56,39 +56,34 @@ public InheritDocTaglet() {
     }
 
     /**
-     * Given an element, a {@code DocTree} in the element's doc comment
-     * replace all occurrences of {@code {@inheritDoc}} with documentation from its
-     * superclass or superinterface.
+     * Given an element and {@code @inheritDoc} tag in that element's doc comment,
+     * returns the (recursive) expansion of that tag.
+     *
+     * This method does not expand all {@code {@inheritDoc}} tags in the given
+     * element's doc comment. To do this, the method must be called for every
+     * such tag.
```

Comment:
This should probably be wrapped in a `<p>` tag:
```suggestion
     * <p>This method does not expand all {@code {@inheritDoc}} tags in the given
     * element's doc comment. To do this, the method must be called for every
     * such tag.</p>
```